### ISSUE TITLE
OpenType fonts no longer showing in fonts list

### ISSUE THREAD

**Issue by orkben** [neetly/figma-agent-linux#179]
I have 2 OpenType fonts and 1 TrueType font in my design work. Recently, the OpenType fonts have stopped working.

The TrueType font still shows up in the list.

I tried installing additional TrueType and OpenType fonts. The TrueType fonts are included in the Figma font list, the OpenType fonts are not.

All of the fonts used in testing above and visible when using LibreOffice Writer.

**Comment by orkben** [comment]
https://github.com/Figma-Linux/figma-linux has a similar problem, but it can see the "Regular" version of the OpenType font, but not the other weights 🤔

**Comment by latin-1** [comment]
Hi, @orkben. Would you mind sending me your font files? Either attach to this issue or send an email.

**Comment by orkben** [comment]
Sure! Thanks for taking a look

The *Nunito* fonts are TTF and are showing in Figma

The *Gilroy* fonts are OTF and aren't showing in Figma

**Comment by orkben** [comment]
I just discovered Nunito is in Google Fonts - so Figma would show it regardless :thinking:

**Comment by orkben** [comment]
Font file attached. This one is visible in Figma even without
figma-agent-linux running it seems

**Comment by latin-1** [comment]
Hello, @orkben. I'm sorry but I did not see your font files. Could you please send those files directly to rev.hikari (at) gmail.com?

**Comment by latin-1** [comment]
Hi, @orkben.

![image](https://github.com/neetly/figma-agent-linux/assets/44946699/e843ba5f-13f0-4e15-87f6-c54c7e719f3f)

I am unable to reproduce this problem on my laptop.

Are you running the latest version of figma-agent-linux? Could you please run the following command and send me the output file?

```sh
curl --header "Origin: https://www.figma.com" "http://127.0.0.1:44950/figma/font-files" > font-files.json
```

Thanks

**Comment by orkben** [comment]
Interesting! Yes, I'm pretty sure I'm on the latest version. I reinstalled
it (disabled the service, removed the files, and ran the install script
again) on the 19 May. I had a look at `~/.local/share/figma-agent`and it
says it was last modified on 19 May.

I've attached the json file.

I'm running OpenSuse Tumbleweed, if that matters.

**Comment by orkben** [comment]
This is the useragent string I've been using in Chrome:
`Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36`

**Comment by orkben** [comment]
[font-files.json](https://github.com/neetly/figma-agent-linux/files/15396370/font-files.json)

**Comment by latin-1** [comment]
It's weird. Could you please go to figma.com, open chrome devtools, open a design file, and check the network panel to see if the request was successfully sent?

![image](https://github.com/neetly/figma-agent-linux/assets/44946699/42913c37-140e-4d90-83bc-73a555551ca9)

(search for `127.0.0.1` or `font-files`)

**Comment by orkben** [comment]
No `font-files` or `127.0.0.1` 

![image](https://github.com/neetly/figma-agent-linux/assets/113874299/542a7401-bed3-472d-bb65-85c0262eb677)

![image](https://github.com/neetly/figma-agent-linux/assets/113874299/d6ea9731-5772-48d3-b9f7-ebe1661f19e1)

**Comment by latin-1** [comment]
Could you please type `navigator.userAgent` in console panel in devtools and see what the result is?

**Comment by orkben** [comment]
Oh dear - I'm sorry. It looks like the user-agent modifier extension I was using had stopped working. How embarrassing 🤦 

I was using [User Agent Switcher](https://chromewebstore.google.com/detail/user-agent-switcher/kchfmpdcejfkipopnolndinkeoipnoia)

I also tried [User-Agent Switcher for Chrome](https://chromewebstore.google.com/detail/djflhoibgkdhkhhcedjiklpkjnoahfmg), which I think the first one was based off of.

The new one that I've now installed and which works is [User-Agent Switcher and Manager](https://chromewebstore.google.com/detail/bhchdcejhohfmigjafbampogmaanbfkg)
